app/tests/src/shelley.js

```diff
@@ -24,9 +24,8 @@ describe('shelley address derivation', () => {
   const mnemonic15Words =
     'abandon abandon abandon abandon abandon abandon abandon abandon abandon abandon abandon abandon abandon abandon address'
   it('should derive base address from 15-words mnemonic', async () => {
-    const cp = await getCryptoProvider(mnemonic15Words, 'mainnet') //TODO: change discriminator to networkId for shelley
+    const cp = await getCryptoProvider(mnemonic15Words, 0) //TODO: change discriminator to networkId for shelley
```

Comment:
Is that `0` the mainnet network id? If so, better reference a named constant here to minimize confusion